public/css/main.css

```diff
@@ -1,13 +1,19 @@
+html {
+	height: 100%;
+}
+
 body {
-	/* margin: 0 auto; */
-	/* width: 600px; */
+	display: flex;
+		align-items: center;
+		justify-content: center;
```

Comment:
No I don't think so. In fact, maybe we should just merge this stuff in now and do the modularity stuff later - we should work together on that!